webui/build.gradle

```diff
@@ -63,6 +63,13 @@ dependencies {
     implementation group: "com.sparkjava", name: "spark-core", version: spark_core_version
 
 
+    compile group: "com.squareup.okhttp3", name: "okhttp", version: okhttp_version
+
+    compile group: 'com.mashape.unirest', name: 'unirest-java', version: unirest_version
+
+    compile group: 'com.opencsv', name: 'opencsv', version: '5.0'
```

Comment:
Replace version with a variable defined in gradle.properties.

Reply:
There was already a variable defined (opencsv_version), but with version 2.3. I changed my code to work with version 2.3. instead.